### DISCUSSION TITLE
Disable showing language server in status bar?

### DISCUSSION THREAD

**Discussion by osa1** [neoclide/coc.nvim#4461] [Q&A]
I just updated coc.nvim and it started to show language server in the status bar, as in the screenshot below. ("rust-analyzer")

![2022-12-30_16-04](https://user-images.githubusercontent.com/448274/210084581-a8000dbd-5ba2-485a-b5e8-42df988fb43c.png)

How do I disable it?

**Comment by fannheyward** [comment]
Check coc-rust-analyzer's doc, there's an option for this. (I'm on mobile so can't post it clearly.

**Reply by osa1** [reply]
Could you let me know which option it is? I checked the list in the README, can't see anything relevant.

**Reply by osa1** [reply]
I found the issue that requested this: https://github.com/fannheyward/coc-rust-analyzer/issues/1065

But still don't know how to disable it. Looking at the code, I don't think it's possible to disable without changing the code.
